tests/test_torch.py

```diff
@@ -219,7 +218,6 @@ def test_sequence_net():
         (97, 999)))
     output.backward(torch.zeros((97, 999)))
     graph = wandb.Graph.transform(graph)
-    pprint(graph)
```

Comment:
Prints in tests are okay because they only actually get printed if the test fails.. unless you really think this is useless

Reply:
its kind of random i guess.